lib/flipper/ui/views/feature.erb

```diff
@@ -289,7 +292,7 @@
             <%== csrf_input_tag %>
             <input type="hidden" id="feature_name" name="_feature" value="<%= feature_name %>">
```

Comment:
I expect that this line can be removed now as it was added when the confirmation text box was put in place. I didn't touch it however as I wanted to avoid unintended side-effects. 

Does anyone have any input on whether this can be removed or if it's used anywhere else? Since it is an input and would get sent back to the server.